arch/arm/soc/atmel_sam/samd/soc_irq.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Interrupt numbers for the Atmel SAMD series.
+ */
+#ifndef _ATMEL_SAM_SAMDSOC_IRQ_H_
+#define _ATMEL_SAM_SAMDSOC_IRQ_H_
+
+#define PM_IRQn 0       /** Power Manager (PM) */
```

Comment:
We do we need this at all, the IRQ for serial should be coming from device tree.

Reply:
Done.